### PR TITLE
Simply can be overused

### PR DESCRIPTION
**Pull request by WaylonWalker**
Simply can make it feel like you have some sort of leg up over those who don't find it simple... words like this can generally be dropped read the same to most folks and not have the connotation.   You can take it, leave it, or implement it yourself, just trying to help out an awesome project, I don't need the credit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,8 +83,7 @@ nbpreview may also be installed via [pip]:
 
 ## Usage
 
-To use nbpreview,
-simply type `nbpreview` into your terminal followed by the path of the notebook you wish to view.
+To use nbpreview, type `nbpreview` into your terminal followed by the path of the notebook you wish to view.
```
Comment (paw-lu) [inline README.md:86]: Can we keep the line break after `nbpreview,`?

```suggestion
To use nbpreview,
type `nbpreview` into your terminal followed by the path of the notebook you wish to view.
```

**Comment by paw-lu** [comment]
This is a good suggestion. Really appreciate it. Bad habit!

Only one small suggestion to keep the line break after the comma and I'll merge.

**Comment by WaylonWalker** [comment]
Updated

**Comment by paw-lu** [comment]
Merged. Thanks for taking the time @WaylonWalker!

**Pull request merged**